client/extensions/wp-job-manager/state/settings/reducer.js

```diff
@@ -39,18 +41,15 @@ export const fetching = createReducer(
  * @param  {Object} action Action object
  * @return {Object} Updated settings
  */
-export const items = createReducer(
-	{},
-	{
-		[ WP_JOB_MANAGER_UPDATE_SETTINGS ]: ( state, { siteId, data } ) => ( {
-			...state,
-			[ siteId ]: data,
-		} ),
-	},
-	itemsSchema
-);
+export const items = ( state = {}, { data, type } ) =>
+	WP_JOB_MANAGER_UPDATE_SETTINGS === type ? data : state;
 
-export default combineReducers( {
-	fetching,
-	items,
-} );
+items.schema = itemsSchema;
```

Comment:
is this schema still valid?
Old reducer schema for items described:
```
{
  [ siteId ]: data 
}
```

In the new version of the items reducer it should be describing:
```
data
```

I'd recommend either applying keyedReducer to each of the reducers to maintain the same shape as before OR to update the schema to include everything (and add tests for it). Something like:


```js
const settingsReducer = keyedReducer(
  'siteId',
  combineReducer({ ... });
)

settingsReducer.schema = moreInclusiveSchema; // but you can just call it schema
```

Then in the test file you wrap the reducer in `state/utils#withSchemaValidation`

Reply:
Fixed by c51f8924db75bb45949d692ad7cfea3bf792189b. I opted to modify the schema to reflect the new `items` subtree shape, since the other subtree in `settings`, `fetching`, shouldn't be serialized anyway.